docs/api.md

```diff
@@ -3658,6 +3661,217 @@ Because this `/#/dl` path returns a web page that causes a file download rather
 
             (html markup data)
 
+
+## OData Dataset Service [/v1/projects/{projectId}/datasets/{name}.svc]
+
+ODK Central presents one OData service for every `Dataset` as a way to get an OData feed of `Entities`. To access the OData service, simply add `.svc` to the resource URL for the given Dataset.
+
++ Parameters
+    + projectId: `7` (number, required) - The numeric ID of the Project
+
+    + `name`: `trees` (string, required) - The `name` of the `Dataset` whose OData service you wish to access.
+
+### Service Document [GET]
+
+The Service Document is essentially the index of all available documents. From this document, you will find links to all other available information in this OData service. In particular, you will find the Metadata Document, as well as a data document for a single `Entities` table defined by the `Dataset`.
```

Comment:
Reading this gave me an impression that this endpoint will give me metadata of all the datasets